src/__test__/components/main/Content.test.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @jest-environment jsdom
+*/
+import React from 'react'
+
+import { render } from '@testing-library/react'
+import Content from '../../../components/main/Content'; 
+import nasaMock from "../../../__mocks__/nasaMock";
+import fetch from 'node-fetch'
+
+beforeAll(() => nasaMock.listen())
+afterAll(() => nasaMock.close())
+
+const responseApi = async () => { 
+    const nasaResponse = await fetch('https://api.nasa.gov/planetary/apod')
+    return  await nasaResponse.json()
+}
+
+describe("Content Component", () => {
+
+    test("must have a image", async () => {
+        
+        
+        const utils = render(<Content data={await responseApi()}/>);
+
+        const image = utils.getByTitle(/A Flag Shaped Aurora over Sweden/i);
+
+        expect(image.src).toContain('https://apod.nasa.gov/apod/image/2103/AuroraFlag_Stalnacke_960.jpg');
+        
+    });
+
+    test("must have explanation", async () => {
+
+        const utils = render(<Content data={await responseApi()}/>);
+
+        const input = utils.getByTestId('description').firstChild
+        
+        expect(input).toBeTruthy()
+
+    });
+
+    test("must have title", async () => {
```

Comment:
the 3 test cases can be merged into one case, it could be called something like 'renders as expected':

```jsx
test("renders as expected", async () => {
  render(<Content data={await responseApi()} />);

  const image = screen.getByTitle(/A Flag Shaped Aurora over Sweden/i);
  const input = screen.getByTestId('description').firstChild
  const title = screen.getByText(/A Flag Shaped Aurora over Sweden/i)

  
  expect(image.src).toContain('https://apod.nasa.gov/apod/image/2103/AuroraFlag_Stalnacke_960.jpg');
  expect(input).toBeTruthy()
  expect(title).toBeInTheDocument()
});
```